typescript/infra/config/environments/mainnet2/agent.ts

```diff
@@ -91,7 +91,7 @@ export const releaseCandidate: AgentConfig = {
       gasPaymentEnforcement: {
         policy: {
           type: GasPaymentEnforcementPolicyType.Minimum,
-          payment: 1,
+          payment: BigInt(1),
```

Comment:
note: BigNumber has been more commonly used but it looks like ethers is moving to BigInt https://github.com/ethers-io/ethers.js/discussions/3040